cmd/monaco/download/download_command.go

```diff
@@ -268,7 +268,7 @@ func printUploadToSameEnvironmentWarning(env manifest.EnvironmentDefinition) {
 		httpClient = clientAuth.NewOAuthClient(context.TODO(), credentials)
 	}
 
-	serverVersion, err = versionClient.GetDynatraceVersion(context.TODO(), rest.NewRestClient(httpClient, rest.CreateRateLimitStrategy()), env.URL.Value)
+	serverVersion, err = versionClient.GetDynatraceVersion(context.TODO(), rest.NewRestClient(httpClient, nil, rest.CreateRateLimitStrategy()), env.URL.Value)
```

Comment:
it was possible to write traffic logs for any command before - with this change, we're losing the option to write logs for downloads I believe. We should not

Reply:
nevermind, read this wrong, this only loses some traffic as do the other commands -> followup as discussed